cmd/konvoy-image-wrapper/cmd/create-package-bundle.go

```diff
@@ -47,6 +47,10 @@ var osToConfig = map[string]OSConfig{
 		configDir:      "bundles/ubuntu20.04",
 		containerImage: "docker.io/library/ubuntu:20.04",
 	},
+	"oracle-9.4": {
+		configDir:      "bundles/oracle9.4",
+		containerImage: "docker.io/library/oraclelinux:9",
```

Comment:
the minor version container images have not been released yet on any registry([ref](https://hub.docker.com/_/oraclelinux/tags?page_size=&ordering=&name=9.)) so using this image which currently points to 9.4 only -

```
[root@8b53e13be278 generated]# cat /etc/oracle-release
Oracle Linux Server release 9.4

[root@8b53e13be278 generated]# cat /etc/redhat-release 
Red Hat Enterprise Linux release 9.4 (Plow)

[root@8b53e13be278 generated]# cat /etc/os-release 
NAME="Oracle Linux Server"
VERSION="9.4"
ID="ol"
ID_LIKE="fedora"
VARIANT="Server"
VARIANT_ID="server"
VERSION_ID="9.4"
PLATFORM_ID="platform:el9"
PRETTY_NAME="Oracle Linux Server 9.4"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:oracle:linux:9:4:server"
HOME_URL="https://linux.oracle.com/"
BUG_REPORT_URL="https://github.com/oracle/oracle-linux"

ORACLE_BUGZILLA_PRODUCT="Oracle Linux 9"
ORACLE_BUGZILLA_PRODUCT_VERSION=9.4
ORACLE_SUPPORT_PRODUCT="Oracle Linux"
ORACLE_SUPPORT_PRODUCT_VERSION=9.4
```

Reply:
we can update the image tag once 9.4 version is released on their official registry/repo -
1. https://container-registry.oracle.com/ords/f?p=113:4:10433988665656:::4:P4_REPOSITORY,AI_REPOSITORY,AI_REPOSITORY_NAME,P4_REPOSITORY_NAME,P4_EULA_ID,P4_BUSINESS_AREA_ID:1,1,Oracle%20Linux,Oracle%20Linux,1,0&cs=3GLgmu4zPDtIjZjF1cXhfpBMVzRl0Sefp1HfIY1Aw88xKXzFyVBlZp58aVGnejQ6Yin9zyBXV3AgvO8P-JhWwgw
2. https://github.com/oracle/container-images/pkgs/container/oraclelinux

Alternatively we can peg image tag to specific SHA but I'm skeptical about reliability of SHA tags to be generally available all time -
<img width="1512" alt="Screenshot 2024-10-03 at 4 16 07 PM" src="https://github.com/user-attachments/assets/89cc11b1-03dc-470c-bceb-c7a7c8c8ae8b">